util/network/src/lib.rs

```diff
@@ -54,6 +54,9 @@
 //! }
 //! ```
 
+//TODO: use Poll from mio
+#![allow(deprecated)]
```

Comment:
Is there a way to do that? Setting an attribute on `extern crate mio` does not work.